src/main/resources/config/liquibase/master.xml

```diff
@@ -88,6 +88,7 @@
     <include file="classpath:config/liquibase/changelog/20240115231508_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20240118175408_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20240307174511_changelog.xml" relativeToChangelogFile="false"/>
+    <include file="classpath:config/liquibase/changelog/20240304125234_changelog.xml" relativeToChangelogFile="false"/>
```

Comment:
```suggestion
    <include file="classpath:config/liquibase/changelog/20240304125234_changelog.xml" relativeToChangelogFile="false"/>
    <include file="classpath:config/liquibase/changelog/20240307174511_changelog.xml" relativeToChangelogFile="false"/>
```
swap for chronological order, changelogs should change separate tables and not affect each other